examples/ecs/bluegreen/taskdef.yaml

```diff
@@ -0,0 +1,21 @@
+family: nginx-service-fam
+executionRoleArn: arn:aws:iam::XXXX:role/ecsTaskExecutionRole
+containerDefinitions:
+  - command: null
+    cpu: 100
+    image: XXXX.dkr.ecr.ap-northeast-1.amazonaws.com/nginx:1
+    memory: 100
+    mountPoints: []
+    name: web
+    portMappings:
+      - containerPort: 80
+compatibilities:
+  - FARGATE
+requiresCompatibilities:
+  - FARGATE
+networkMode: awsvpc
+memory: 512
+cpu: 256
+pidMode: ""
+status: ACTIVE
```

Comment:
Is this `status` field can be specified by the user?

Reply:
Yes, it is 🙏 
```go
	// The status of the task definition.	
	Status TaskDefinitionStatus
```
ref: https://github.com/aws/aws-sdk-go-v2/blob/main/service/ecs/types/types.go#L3308-L3309

Reply:
Oops, I get your point 👍 We're not going to use this anywhere else while registered this TaskDefinition 🙆‍♀️ Will remove it from this sample 👍

Reply:
addressed on https://github.com/pipe-cd/pipe/pull/2079/commits/fd360ed6ee4989ef6e41ce9e71630531b7c3ac8b 🙏

Reply:
👍